tests/src/test/scala/cats/tests/NestedTests.scala

```diff
@@ -18,7 +18,7 @@ class NestedTests extends CatsSuite {
 
   {
     // Invariant composition
-    implicit val instance = ListWrapper.invariant
+    implicit val instance = Nested.catsDataInvariantForNested(ListWrapper.invariant, ListWrapper.invariant)
```

Comment:
I'll check again, I had a coverage drop which was fixed by this but I think there was an underlying problem I fixed.